### ISSUE TITLE
Can Not Pass Event SignChangeEvent

### ISSUE THREAD

**Issue by Stuart74** [CyberdyneCC/Thermos#564]
You will receive NO SUPPORT if you do not follow the template below. (Vous ne receverez pas aucune aide si vous effacer ce modèle)

**Server Log:** *([Github Gist](https://gist.github.com/Stuart74/3570b955813c91f320792faa2a5112ec))*

**FML Log(s):** *([Github Gist](https://gist.github.com/Stuart74/bac3295547d78932da80f27cb17fc6c2))*

**Explanation of issue:**
When trying to create a shop using this plugin [Shop Plugin](https://www.spigotmc.org/resources/shop-a-simple-intuitive-shop-plugin.9628/) the sign is supposed to attach to the chest and change details accordingly set in the shops plugin config but each time i try nothing occurs and get the Can Not Pass Event SignChangeEvent errors in console including the errors for the same event on startup of the server.

**How to recreate this issue:**
Try to create a shop it will error everytime you use the sign

**Modpack Name:** *Bevos Tech Pack Reloaded*
[ModList](https://www.modpackindex.com/modpack/1-7-10/bevos-tech-pack)

**Mods Installed:** 

**Plugins Installed:**
[Plugins](http://puu.sh/s0sh8.png)

**Warmroast Report:** *(Optional)*

**Thermos Version:** 
 1.7.10-R0.1-SNAPSHOT | 1.7.10-1614.58 (MC: 1.7.10)

**Forge Version:**
1614

**Comment by Time6628** [comment]
Have you tried contacting the plugin author?

**Comment by Stuart74** [comment]
Yes the plugin author is going to put a fix in place but I still feel its an issue on thermos side as when i try the same thing with a pure vanilla 1.8 spigot build it works as intended.

**Comment by spannerman79** [comment]
> I still feel its an issue on thermos side as when i try the same thing with a pure vanilla 1.8 spigot build it works as intended

Wrong version to test against. I'm going to assume the bukkit API changed between 1.7.10 (as that is what Thermos is based on) and 1.8.  Should test with vanilla Spigot 1.7.10 instead.
